cmd/agent/app/status.go

```diff
@@ -63,6 +63,11 @@ var statusCmd = &cobra.Command{
 			return err
 		}
 
+		if err = common.SetupSystemProbeConfig(sysProbeConfFilePath); err != nil {
+			// log.Infof("System probe config not found, disabling pulling system probe info in the status page: %v", err)
```

Comment:
It is strange the log functions don't work here, considering logging is setup just above this. Regardless, I don't think the `status` command needs to be logging here, since it could be confusing if those were coming from the process running the `status` command or the agent itself.

I think you could ignore the error altogether, with:
`_ = common.SetupSystemProbeConfig(sysProbeConfFilePath)`